src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -107,24 +111,64 @@
 (defn notifications-list
   []
   (let [notifications (<sub [:activity-center/notifications-per-read-status])]
-    [rn/flat-list {:style          {:padding-horizontal 8}
-                   :data           notifications
+    [rn/flat-list {:data           notifications
                    :key-fn         :id
                    :on-end-reached #(>evt [:activity-center.notifications/fetch-next-page])
                    :render-fn      render-notification}]))
 
+(defn filter-selector-read []
+  (let [unread-filter-enabled?  (<sub [:activity-center/status-filter-unread-enabled?])]
+    ;; TODO: Replace the button by a Filter Selector component once available for use.
+    [button/button {:icon     true
+                    :type     (if unread-filter-enabled? :primary :outline)
+                    :size     32
+                    :on-press #(if unread-filter-enabled?
+                                 (>evt [:activity-center.notifications/fetch-first-page {:status-filter :read}])
+                                 (>evt [:activity-center.notifications/fetch-first-page {:status-filter :unread}]))}
+     :unread]))
+
 (defn activity-center []
   (reagent/create-class
    {:component-did-mount #(>evt [:activity-center.notifications/fetch-first-page {:status-filter :unread}])
     :reagent-render
     (fn []
-      [:<>
-       [topbar/topbar {:navigation {:on-press #(>evt [:navigate-back])}
-                       :title      (i18n/label :t/activity)}]
-       ;; TODO(ilmotta): Temporary solution to switch between read/unread
-       ;; notifications while the Design team works on the mockups.
-       [button/button {:on-press #(>evt [:activity-center.notifications/fetch-first-page {:status-filter :unread}])}
-        "Unread"]
-       [button/button {:on-press #(>evt [:activity-center.notifications/fetch-first-page {:status-filter :read}])}
-        "Read"]
-       [notifications-list]])}))
+      (let [screen-padding 20]
+        [:<>
+         [button/button {:icon     true
+                         :type     :grey
+                         :size     32
+                         :style    {:margin-vertical 12
+                                    :margin-left     screen-padding}
+                         :on-press #(>evt [:navigate-back])}
+          :close]
```

Comment:
so we could distinguish them in the code

Reply:
Which namespace should we use? I saw different names being used, like `:main-icons2`. Should I use `:main-icons` instead (without the number 2)?

Reply:
`:main-icons2` if you use quo2 component

Reply:
I'm marking this thread as resolved because I implemented this in another branch I'm currently working on.

d0adb04f993a140fb46435d878d07945b5a10e2b